src/lib.rs

```diff
@@ -212,6 +212,17 @@ impl CommitDomain {
         }
     }
 
+    /// Constructs a new `CommitDomain` from different values for `hash_domain` and `blind_domain`
```

Comment:
```suggestion
    /// Constructs a new `CommitDomain` from different values for `hash_domain` and `blind_domain`.
```

This needs documentation indicating how this is intended to be used (ideally with a specification link, e.g. to how it is used in ZSAs).

Reply:
Done